finality-aleph/src/network/substrate.rs

```diff
@@ -22,6 +36,15 @@ const AUTHENTICATION_PROTOCOL_NAME: &str = "/auth/0";
 /// Name of the network protocol used by Aleph Zero to synchronize the block state.
 const BLOCK_SYNC_PROTOCOL_NAME: &str = "/sync/0";
 
+/// Protocols used by the network.
+#[derive(Debug, PartialEq, Eq, Copy, Clone, Hash)]
+pub enum Protocol {
```

Comment:
This is technically not strictly substrate and perhaps would be better in `mod.rs`. Not insisting though.

Hmm~, alternatively... is this even needed now? I.e. does any other component now use bare `Protocol`s rather than just `ProtocolNaming`, in which case it would be sufficient to only have it behave reasonably with constant strings it contains? On a quick glance this might be the case, but I'm not completely sure.

Reply:
annihilated